00_Download_Script.bat

```diff
@@ -1,34 +1,91 @@
-@echo off
-rem 00_Download_Script.bat
-rem http://www.neko.ne.jp/~freewing/software/build_fritzing_100_windows/
-rem Copyright (c) 2023 FREE WING,Y.Sakamoto
-echo %0
-@echo off
-
```

Comment:
There are “diffs” in unchanged sections and files.
(probably due to line break code changes)

![image](https://github.com/user-attachments/assets/7ad21aa6-7c23-49e1-9d67-7aee6d40c88b)